OpenRA.Mods.Common/Widgets/ProductionTabsWidget.cs

```diff
@@ -87,6 +87,9 @@ public class ProductionTabsWidget : Widget
 		Lazy<ProductionPaletteWidget> paletteWidget;
 		string queueGroup;
 
+		public string ClickSound = ChromeMetrics.Get<string>("ClickSound");
```

Comment:
Please make these `readonly` and put them up top with the other fields that are exposed to yaml.